.github/workflows/ci_test-tpu.yml

```diff
@@ -1,96 +1,104 @@
 name: TPU tests
 
+# Workflow Steps:
+#  1. Setup gcloud CLI
+#  2. Checkout Pytorch Lightning
+#  3. Set up Python
+#  4. Set up Python 3.7
+#  5. Get Current PR Number
+#  6. Checkout ml-testing-accelerators
+#  7. Install jsonnet
+#  8. Get Cluster Credentials
+#  9. Deploy the job on the kubernetes cluster
+#  10. Statistics
+#  11. Upload coverage results
+#  12. Upload coverage to Codecov
+
 on:
   push:
     branches: [master, "release/*"]
-# TODO: temporal disable TPU testing until we find way how to pass credentials to forked PRs
-#  pull_request:
-#    branches:
-#      - master
 
 env:
   GKE_CLUSTER: lightning-cluster
   GKE_ZONE: us-central1-a
-  IMAGE: gcr.io/${{ secrets.GKE_PROJECT }}/tpu-testing-image
   MAX_CHECKS: 360
   CHECK_SPEEP: 5
 
 jobs:
-  setup-build-publish-deploy:
+  setup-deploy:
     name: tpu-testing-job
     runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
       matrix:
-        python-version: [3.6, 3.7]
+        python-version: [3.6]
```

Comment:
why not 3.7?

Reply:
I am putting it back, but running a matrix of 4 seems quite a lot to me.